intake-catalogs/hydro/camels.yaml

```diff
@@ -19,8 +19,29 @@ sources:
         parse_dates: {'date': ['Year', 'Mnth', 'Day', 'Hr']}
     metadata:
       origin_url: 'https://ral.ucar.edu/solutions/products/camels'
+      
+  basin_mean_forcing_aws:
+    description: Camels Basin Mean Forcing
+    parameters: # User parameters
+      basin_id:
+        description: Basin ID
+        type: str
+        default: "05056000"
+      source:
+        description: Source dataset
+        type: str
+        default: daymet
+    driver: csv
+    args:
+      urlpath: 's3://pangeo-camels/basin_timeseries_v1p2_metForcing_obsFlow/basin_dataset_public_v1p2/basin_mean_forcing/{{ source }}/*/{{ basin_id }}_*.txt'
+      csv_kwargs:
+        sep: '\s+'
+        header: 3
+        parse_dates: {'date': ['Year', 'Mnth', 'Day', 'Hr']}
+    metadata:
+      origin_url: 'https://ral.ucar.edu/solutions/products/camels'
 
-  usgs_streamflow:
+  usgs_streamflow_gcp:
```

Comment:
Should we worry about breaking user code when we rename things? Can we have aliases?

Reply:
I don't think that is necessary just yet. These existing entries just went in so I doubt there are any users at this point (except for me).